content/07.pathogenesis.md

```diff
@@ -10,12 +10,20 @@ This section would also be great for the introduction of zoonotic diseases which
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
 
-#### Viral Replication, Spreading and Transmission
+#### Viral Replication and Spreading
 
+[Renamed section with transmission dynamics below] 
 [Basic introduction into replication cycle]
-[What is the basic reproductive rate]
 [What are the routes of transmission]
 
+#### Reproduction Number and Dynamics of Transmission
+
+Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions. The basic reproduction number, R0, is the expected number of new infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. The effective reproduction number, R(t), describes how the reproduction number may change over time, and is used to quantify deviations in R from R0, for example as some fraction of the population becomes infected, or as interventions are put into place. R0 and R(t) can be estimated directly from epidemiological data or inferred using mathematical modeling, the latter is typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74]. 
+
+R0 for COVID-19 is estimated to lie in the range R0=1.4-6.5 [@doi:10.1016/S0140-6736(20)30260-9; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used. Most estimates currently derive from populations in Asia, since outbreaks in Europe and North America are more recent. Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R0. For data-derived estimates, in one study of international cases, the predicted value is R0=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R0=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10.1016/S0140-6736(20)30260-9], and on a cruise ship where an outbreak occurred, predicted R0=2.28 [@doi:10.1016/j.ijid.2020.02.033]. SIR model-derived estimates of R0 range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549], R0=4.86 in Iran [@doi:10.1101/2020.03.20.20038422], and R0=4.8 in France [@doi:10.1101/2020.03.22.20040915]. Using the same model as for the French population, this study estimated R0=2.6 in South Korea, which is consistent with other studies [@url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf]. From a meta-analysis of studies estimating R0, [@doi:10.1093/jtm/taaa021] predict the median as R0=2.79. 
```

Comment:
```suggestion
R0 for COVID-19 is estimated to lie in the range R0=1.4-6.5 [@doi:10.1016/S0140-6736(20)30260-9; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221]; estimates vary considerably depending on the data and the methods used. 
Most estimates currently derive from populations in Asia, since outbreaks in Europe and North America are more recent. 
Data-derived estimates (i.e. those that do not incorporate SIR-type models into their analysis) typically predict lower values of R0. 
For data-derived estimates, in one study of international cases, the predicted value is R0=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R0=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10.1016/S0140-6736(20)30260-9], and on a cruise ship where an outbreak occurred, predicted R0=2.28 [@doi:10.1016/j.ijid.2020.02.033]. 
SIR model-derived estimates of R0 range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549], R0=4.86 in Iran [@doi:10.1101/2020.03.20.20038422], and R0=4.8 in France [@doi:10.1101/2020.03.22.20040915]. 
Using the same model as for the French population, this study estimated R0=2.6 in South Korea, which is consistent with other studies [@url:https://github.com/parksw3/Korea-analysis/blob/master/v1/korea.pdf]. 
From a meta-analysis of studies estimating R0, [@doi:10.1093/jtm/taaa021] predict the median as R0=2.79. 
```